### PR TITLE
UI Cleanup on Taxonomy Displays

### PR DESCRIPTION
**Pull request by jeznorth**
## Links to Jira Tickets

- [SIMSBIOHUB-479](https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-479)

## Description of Changes

- Updating species / taxonomy display

### CODE REVIEW

File: app/src/components/species/components/SpeciesCard.tsx [hunk 1]
```diff
@@ -1,23 +1,44 @@
...
+            </Typography>
+            &nbsp;(<em>{props.scientificName}</em>)
+          </>
+        ) : (
+          <em>{props.scientificName}</em>
+        )}
+      </Typography>
+      <Chip
+        title="Taxonomic serial number"
+        label={'TSN:' + props.tsn}
```
Comment (curtisupshall) [inline app/src/components/species/components/SpeciesCard.tsx:35]: Nit: use template string. `props.tsn` is probably type `number`, and `string + number` is very unpredictable in JS.

**Review by NickPhura — DISMISSED** [review]
🦖

**Review by NickPhura — COMMENTED** [review]
🦕

**Review by al-rosenthal — APPROVED** [review]
🦕

File: app/src/components/species/components/SpeciesCard.tsx (hunk 1 above)
Comment (jeznorth) [reply app/src/components/species/components/SpeciesCard.tsx:35]: Will fix via another branch.

**Review by NickPhura — APPROVED** [review]
🐞

**Pull request merged**